R/ci.R

```diff
@@ -994,6 +994,19 @@ ci.meta_bma <- ci.meta_random
 
 
 
+# lmtest::coeftest -------------------------------------------------------
+
+#' @export
+ci.coeftest <- function(model, ci=0.95, vcov.=NULL, df=NULL, ...) {
```

Comment:
I think you can remove the arguments `vcov` and `df` here, as these are only used inside `coefci()`, while `confint()` is applied to the coeftest-object that already has processed the `vcov`-argument.